client/src/App.test.js

```diff
@@ -1,8 +1,6 @@
-import { render, screen } from '@testing-library/react';
+import { render } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
+test('renders App component', () => {
   render(<App />);
```

Comment:
Even though you have a test file would it be beneficial to add a comment to say the test passed here? For Stem review purposes?

Reply:
Great idea, thank you Samelia!